tpl/tplimpl/tplimpl_integration_test.go

```diff
@@ -584,3 +584,19 @@ title: p5
 		`<meta name="twitter:description" content="m n and **o** can&#39;t.">`,
 	)
 }
+
+func TestCommentShortcode(t *testing.T) {
+	t.Parallel()
+
+	files := `
+-- hugo.toml --
+disableKinds = ['page','rss','section','sitemap','taxonomy','term']
+-- layouts/index.html --
+{{ .Content }}
+-- content/_index.md --
+a{{< comment >}}b{{< /comment >}}c
```

Comment:
This is fine for the test, but in the docs/examples I would recommend that we use the `{{%` separator. It would make not no difference for the end result, but would save a few CPU cycles ...

Reply:
Understood. I'll leave the test as-is.